config/crd/experimental/gateway.networking.k8s.io_httproutes.yaml

```diff
@@ -1692,6 +1692,26 @@ spec:
                         type: object
                       maxItems: 8
                       type: array
+                    timeouts:
+                      description: "Timeouts defines the timeouts that can be configured
+                        for an HTTP request. \n Support: Core \n <gateway:experimental>"
+                      properties:
+                        backendRequest:
+                          description: "BackendRequest specifies a timeout for an
+                            individual request from the gateway to a backend service.
+                            Default is the value of Request timeout. \n Support: Extended"
```

Comment:
This needs another regen because not up to date with the comment in the go code?

Reply:
addressed in 2678d3cf1fa6774a4a405583e1d9e528f0a66209